### PR TITLE
Bugfix: Tabs content validation errors are not displayed on submit

### PR DESCRIPTION
**Pull request by Sipaha**
Issue: https://github.com/formio/formio.js/issues/1249

### CODE REVIEW

File: src/components/tabs/Tabs.js [hunk 1]
```diff
@@ -55,44 +53,6 @@ export default class TabsComponent extends NestedComponent {
     return schema;
   }
 
-  build(state, showLabel) {
-    if (this.options.flatten) {
```
Comment (travist) [inline src/components/tabs/Tabs.js]: Why was this logic removed? We need this for PDF generation.

Comment (Sipaha) [reply src/components/tabs/Tabs.js]: Sorry. This fix was initially for our project where this logic doesn't need. Fixed it

File: src/components/tabs/Tabs.js [hunk 2]
```diff
@@ -188,56 +189,6 @@ export default class TabsComponent extends NestedComponent {
...
       .addClass(this.tabLinks[index].tabLink, 'active')
       .addClass(this.tabs[index], 'active');
-
-    this.triggerChange();
-  }
-
-  /**
-   * Return all the components within all the tabs.
-   */
-  getAllComponents() {
```
Comment (travist) [inline src/components/tabs/Tabs.js]: Why did you remove all this code? This is needed to validate the items within each tab.

Comment (Sipaha) [reply src/components/tabs/Tabs.js]: This code is not required when we render all components from all tabs on the build stage. Validity checking from NestedComponent, in this case, works well.

Comment (travist) [reply src/components/tabs/Tabs.js]: Well, we need this code back for us to accept this pull request. Other people use this renderer and we cannot just rip out code because it does not fit your use case.

Comment (Sipaha) [reply src/components/tabs/Tabs.js]: This code is the main problem which this pull request will fix and component behaviour won't change for any users (not only for us). Do you have any auto tests or testers who can verify that nothing changed from a user side? Or you need a deep analysis of the solved problem to accept this request? Removing the build method was a mistake, but this code (getAllComponents and checkValidity) just not working and leads to a lot of bugs. Any attempts to correct this logic will be failed in my opinion.

Comment (travist) [reply src/components/tabs/Tabs.js]: Ok, thanks for the response. We will take a look as soon as we can if your changes work for validation across tabs.

**Comment by travist** [comment]
Please rebase and resubmit if you still need this.
